python/doc/usecases/use_case_cantilever_beam.rst

```diff
@@ -42,15 +44,18 @@ We assume that the random variables E, F, L and I are dependent and associated w
         \end{pmatrix}
 
 
-In other words, we consider that the variables L and I are negatively correlated : when the length L increases, the moment of intertia I decreases.
+In other words, we consider that the variables :math:`L` and :math:`I` are negatively correlated : when the length :math:`L` increases, the moment of intertia :math:`I` decreases.
```

Comment:
```suggestion
In other words, we consider that the variables :math:`L` and :math:`I` are negatively correlated: when the length :math:`L` increases, the moment of inertia :math:`I` decreases.
```

Reply:
I do not see the change in the page, or is it?

Reply:
![image](https://user-images.githubusercontent.com/31351465/235299722-6731df19-7d96-446a-90c0-4ddc5d31a0a1.png)

Reply:
yes...